SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+The Drand team will be actively supporting the most recent minor version of drand.
+Upcoming major or minor releases will be announced with a prerelease at least one week in advance, and will be communicated through the leagueofentropy@googlegroups.com email list.
```

Comment:
Add link to sign up?

Reply:
https://groups.google.com/forum/#!forum/leagueofentropy appears to be visible archives, but not open signup. @daviddias  is there a better place for people to be notified? maybe just subscribe to notifications of releases on the github repo? we can't do that in advance though.

Reply:
I believe we shouldn't rely on the LoE mlist as the primary channel. See my comment at https://github.com/drand/drand/pull/366#discussion_r429782575